app/views/account/pref.scala

```diff
@@ -14,7 +14,7 @@ object pref:
   private def categFieldset(categ: lila.pref.PrefCateg, active: lila.pref.PrefCateg) =
     div(cls := List("none" -> (categ != active)))
 
-  private def setting(name: Frag, body: Frag) = st.section(h2(name), body)
+  private def setting(name: Frag, body: Frag, settingid: String) = st.section(a(href := "#" + settingid)(h2(id := settingid)(name)), body)
```

Comment:
coding standards `settingId`

Reply:
changed, when I started, I was undecided whether to leave it like that or not